docs/en/query_language/functions/date_time_functions.md

```diff
@@ -169,6 +169,77 @@ Converts a date or date with time to a UInt16 number containing the ISO Year num
 
 Converts a date or date with time to a UInt8 number containing the ISO Week number.
 
+## toCustomYear
+
+Converts a date or date with time to a UInt16 number containing the Custom Year number.
+```
+toCustomYear(DateTime, [, CustomDate][, Timezone])
+```
+
+## week(date[,mode])
+This function returns the week number for date or datetime. The two-argument form of week() enables you to specify whether the week starts on Sunday or Monday and whether the return value should be in the range from 0 to 53 or from 1 to 53. If the mode argument is omitted, the default mode is 0.
+The following table describes how the mode argument works.
+
+| Mode | First day of week | Range |  Week 1 is the first week … |
+| ----------- | -------- | -------- | ------------------ |
+|0|Sunday|0-53|with a Sunday in this year
+|1|Monday|0-53|with 4 or more days this year
+|2|Sunday|1-53|with a Sunday in this year
+|3|Monday|1-53|with 4 or more days this year
+|4|Sunday|0-53|with 4 or more days this year
+|5|Monday|0-53|with a Monday in this year
+|6|Sunday|1-53|with 4 or more days this year
+|7|Monday|1-53|with a Monday in this year
+|8|Sunday|1-53|contains January 1
+|9|Monday|1-53|contains January 1
+
+For mode values with a meaning of “with 4 or more days this year,” weeks are numbered according to ISO 8601:1988:
+
+- If the week containing January 1 has 4 or more days in the new year, it is week 1.
+
+- Otherwise, it is the last week of the previous year, and the next week is week 1.
+
+For mode values with a meaning of “contains January 1”, the week contains January 1 is week 1. It doesn't matter how many days in the new year the week contained, even if it contained only one day.
+
+```
+week(DateTime1, [, mode][, Timezone])
```

Comment:
Too complicated name for the documentation. Please rename `DateTime1` => `date`

Reply:
ok